include/staff/slaplans.inc.php

```diff
@@ -36,7 +36,7 @@
 
 $pageNav = new Pagenate($count, $page, PAGE_LIMIT);
 $pageNav->setURL('slas.php', $qs);
-$showing = $pageNav->showing().' '._N('SLA plan', 'SLA plans', $count);
+$showing = $pageNav->showing().' '._N('Service Level Agreement', 'Service Level Agreements', $count);
```

Comment:
You changed "SLA Plan" / "SLA Plans" to the full length "Service Level Agreement" / "Service Level Agreements" here and in include/class.slas.php, but not in scp/slas.php. Is this for a specific reason or not? If not, you should may change it in scp/slas.php too ;)